doc/user/content/security/self-managed/authentication.md

```diff
@@ -82,6 +82,71 @@ users:
 [^1]: The `mz_system` user is also used by the Materialize Operator for upgrades
 and maintenance tasks.
 
+## Configuring SASL/SCRAM authentication
+
+{{< note >}}
+SASL/SCRAM-SHA-256 authentication requires Materialize v0.147.16 or later.
+{{</ note >}}
+
+SASL/SCRAM-SHA-256 authentication is a challenge-response authentication mechanism
+that provides security for PostgreSQL wire protocol connections. It is
+compatible with PostgreSQL clients that support SCRAM-SHA-256.
+
+To configure Self-Managed Materialize for SASL/SCRAM authentication:
+
+| Configuration | Description
+|---------------| ------------
+|`spec.authenticatorKind` | Set to `Sasl` to enable SASL/SCRAM-SHA-256 authentication for PostgreSQL connections.
+|`external_login_password_mz_system` | To the Kubernetes Secret referenced by `spec.backendSecretName`, add the secret key `external_login_password_mz_system`. This is the password for the `mz_system` user [^1], who is the only user initially available when SASL authentication is enabled.
+
+For example, if using Kind, in the `sample-materialize.yaml` file:
+
+```
```

Comment:
heh ... I know the css for highlighting rows is wonky in dark mode, but could we use the `hc {hl_lines="14 24"}` (guessing the line numbers are the same as the password auth yaml's)